apps/dotcom/client/src/tla/components/TlaFileError/TlaFileError.tsx

```diff
@@ -0,0 +1,79 @@
+import { TLRemoteSyncError, TLSyncErrorCloseEventReason } from '@tldraw/sync-core'
+import { ReactElement, useEffect } from 'react'
+import { sadFaceIcon } from '../../../components/ErrorPage/ErrorPage'
+import { useSetIsReady } from '../../hooks/useIsReady'
+import { TlaSignInButton } from '../TlaSignInButton/TlaSignInButton'
+import styles from './TlaFileError.module.css'
+
+function DefaultError() {
+	return (
+		<TlaFileErrorContent
+			header="Something went wrong"
+			para1="Please try refreshing the page."
+			para2="Still having trouble? Let us know at hello@tldraw.com"
+		/>
+	)
+}
+
+export function TlaFileError({ error }: { error: unknown }) {
+	const setIsReady = useSetIsReady()
+	useEffect(() => {
+		setIsReady()
+	}, [setIsReady])
+
+	if (!(error instanceof TLRemoteSyncError)) return <DefaultError />
+
+	switch (error.reason) {
+		case TLSyncErrorCloseEventReason.NOT_FOUND: {
+			return (
+				<TlaFileErrorContent
+					header="Not found"
+					para1="The file you are looking for does not exist."
+				/>
+			)
+		}
+		case TLSyncErrorCloseEventReason.NOT_AUTHENTICATED: {
+			return (
+				<TlaFileErrorContent
+					header="Private file"
```

Comment:
I'll be nitpicky about this:
- "private file" might be misleading to the owner of the file. they might get to this error msg and think that a _shared_ (invite-only) file they have is actually private to just them instead of being shared with multiple people. we should reserve the word "private" for things that are truly only viewable by one person to avoid confusion.

here's some other apps for inspiration:
<img width="504" alt="Screenshot 2024-10-28 at 14 02 53" src="https://github.com/user-attachments/assets/ce0c38b1-407b-4228-9a82-e5e555bc1f61">

<img width="675" alt="Screenshot 2024-10-28 at 14 05 22" src="https://github.com/user-attachments/assets/ed2fb28f-3dd0-4431-8582-51d14d0b582d">


<img width="1253" alt="Screenshot 2024-10-28 at 14 02 29" src="https://github.com/user-attachments/assets/1b9e7276-21a3-4b9d-8705-7536727907cf">